lib/chef/knife/ec_backup.rb

```diff
@@ -167,76 +169,84 @@ def download_org_invitations(name)
       end
 
       def ensure_dir(dir)
-        if !File.exist?(dir)
-          FileUtils.mkdir_p(dir)
-        end
+        FileUtils.mkdir_p(dir) unless File.exist?(dir)
       end
 
-      PATHS = %w(chef_repo_path cookbook_path environment_path data_bag_path role_path node_path client_path acl_path group_path container_path)
+      PATHS = %w[chef_repo_path cookbook_path environment_path data_bag_path role_path node_path client_path acl_path group_path container_path].freeze
       def download_org_data(name)
         old_config = Chef::Config.save
 
         begin
-          # Clear out paths
-          PATHS.each do |path|
-            Chef::Config.delete(path.to_sym)
-          end
-          Chef::Config.chef_repo_path = "#{dest_dir}/organizations/#{name}"
-          Chef::Config.versioned_cookbooks = true
+      # Clear out paths
+      PATHS.each do |path|
+        Chef::Config.delete(path.to_sym)
+      end
+      Chef::Config.chef_repo_path = "#{dest_dir}/organizations/#{name}"
+      Chef::Config.versioned_cookbooks = true
 
-          Chef::Config.chef_server_url = "#{server.root_url}/organizations/#{name}"
+      Chef::Config.chef_server_url = "#{server.root_url}/organizations/#{name}"
 
-          ensure_dir(Chef::Config.chef_repo_path)
+      ensure_dir(Chef::Config.chef_repo_path)
 
-          # Download the billing-admins, public_key_read_access ACL and group as pivotal
-          chef_fs_config = Chef::ChefFS::Config.new
+      # Download the billing-admins, public_key_read_access ACL and group as pivotal
+      chef_fs_config = Chef::ChefFS::Config.new
 
-          paths = ['/acls/groups/billing-admins.json', '/groups/billing-admins.json', '/groups/admins.json']
-          paths.push('/acls/groups/public_key_read_access.json', '/groups/public_key_read_access.json') if server.supports_public_key_read_access?
+      unless skip_objects.include? 'groups'
+        paths = ['/acls/groups/billing-admins.json', '/groups/billing-admins.json', '/groups/admins.json']
+        paths.push('/acls/groups/public_key_read_access.json', '/groups/public_key_read_access.json') if server.supports_public_key_read_access?
 
```

Comment:
🤔 I don't think these format adjustments are right. Everything is now at the same level as the function `download_org_data` on line 176